examples/postgres/docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3.7"
+services:
+
+  proxy:
+    image: envoyproxy/envoy-dev:latest
+    volumes:
+      - ./envoy.yaml:/etc/envoy.yaml
+    networks:
+      envoymesh:
+        aliases:
+          - envoy
+    expose:
```

Comment:
this expose isnt necessary as ports below will expose the ports

Reply:
Fixed!